mission-board/src/main/java/com/board/comment/controller/CommentController.java

```diff
@@ -0,0 +1,60 @@
+package com.board.comment.controller;
+
+import com.board.comment.controller.dto.CommentRequest;
+import com.board.comment.controller.dto.CommentResponse;
+import com.board.comment.controller.dto.CommentResponses;
+import com.board.comment.mapper.CommentMapper;
+import com.board.comment.service.CommentService;
+import com.board.login.annotation.Login;
+import com.board.member.domain.Member;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+@Slf4j
+public class CommentController {
+
+    private final CommentService commentService;
+
+    @PostMapping("/articles/{articleId}/comments")
+    public ResponseEntity<CommentResponse> createComment(@RequestBody CommentRequest commentRequest, @PathVariable Long articleId, @Login Member member) {
+        CommentResponse commentResponse = CommentMapper.toCommentResponse(commentService.createComment(commentRequest, member, articleId));
+        URI location = URI.create("/api/articles/" + commentResponse.articleId());
+        log.info("{}번 게시물, {}님 댓글 작성 완료. [내용]: {}", commentResponse.articleId(), commentResponse.memberNickName(), commentResponse.content());
+        return ResponseEntity.created(location).build();
+    }
+
+    @GetMapping("/articles/{articleId}/comments")
+    public ResponseEntity<CommentResponses> showAllComments(@PathVariable Long articleId) {
+        CommentResponses commentResponses = CommentMapper.toCommentResponses(commentService.showAllComments(articleId));
+        log.info("게시물에 대한 모든 댓글 조회 완료.");
+        return ResponseEntity.ok(commentResponses);
+    }
+
+    @PatchMapping("/comments/{commentId}")
+    public ResponseEntity<CommentResponse> updateComment(@PathVariable Long commentId, @Login Member member, @RequestBody CommentRequest commentRequest) {
+        CommentResponse commentResponse = CommentMapper.toCommentResponse(commentService.updateComment(commentRequest, member, commentId));
+        log.info("{}님 댓글 수정 완료. [내용]: {}", commentResponse.memberNickName(), commentResponse.content());
+        return ResponseEntity.ok(commentResponse);
+    }
+
+    @DeleteMapping("/comments/{commentId}")
+    public ResponseEntity<CommentResponse> deleteComment(@PathVariable Long commentId, @Login Member member) {
+        CommentResponse commentResponse = CommentMapper.toCommentResponse(commentService.deleteComment(member, commentId));
+        log.info("{}님 댓글 삭제 완료", commentResponse.memberNickName());
+        return ResponseEntity.ok(commentResponse);
+    }
+
+    @GetMapping("/members/comments")
```

Comment:
자원을 표기하려면
```suggestion
    @GetMapping("/members/me/comments")
```

이런식으로 해도 좋을 것 같습니다 ㅎㅎ

Reply:
me 가 쓰일 수 있다는 것은 처음 알았습니다 감사합니다!